hollow-explorer-ui/src/main/java/com/netflix/hollow/explorer/ui/HollowExplorerUI.java

```diff
@@ -105,13 +108,21 @@ public HollowReadStateEngine getStateEngine() {
             return client.getStateEngine();
         return stateEngine;
     }
-    
+
+    public String getHeaderDisplayString() {
+        return headerDisplayMap.get("headerDisplayString");
```

Comment:
minor: might be good to move this to a private static final constant

Reply:
done